ng2-components/ng2-activiti-form/src/components/widgets/amount/amount.widget.spec.ts

```diff
@@ -17,13 +17,42 @@
 
 import { AmountWidget } from './amount.widget';
 import { FormFieldModel } from './../core/form-field.model';
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+import { AlfrescoAuthenticationService, AlfrescoSettingsService, AlfrescoApiService, CoreModule } from 'ng2-alfresco-core';
+import { FormService } from './../../../services/form.service';
+import { EcmModelService } from './../../../services/ecm-model.service';
+import { ActivitiAlfrescoContentService } from '../../../services/activiti-alfresco.service';
 
 describe('AmountWidget', () => {
 
     let widget: AmountWidget;
+    let fixture: ComponentFixture<AmountWidget>;
+    let debug: DebugElement;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            imports: [
+                CoreModule
```

Comment:
If you use `CoreModule.forRoot()` you can remove all Core providers from the `providers` section and imports section.